api/DeviceMotionEventAcceleration.json

```diff
@@ -23,10 +23,14 @@
             "version_removed": "79"
           },
           "firefox": {
-            "version_added": null
+            "version_added": "6",
+            "partial_implementation": true,
+            "notes": "Doesn't expose the <code>DeviceMotionEventAcceleration</code> interface itself."
           },
           "firefox_android": {
-            "version_added": null
+            "version_added": "6",
+            "partial_implementation": true,
+            "notes": "Doesn't expose the <code>DeviceMotionEventAcceleration</code> interface itself."
```

Comment:
Let's have the note only on the interface entry itself, not on any of the members.